core/templates/components/shared-component.module.ts

```diff
@@ -26,14 +26,14 @@ import { AngularFireAuth, AngularFireAuthModule, USE_EMULATOR } from '@angular/f
 import { CustomFormsComponentsModule } from './forms/custom-forms-directives/custom-form-components.module';
 import { DynamicContentModule } from './angular-html-bind/dynamic-content.module';
 import { FormsModule, ReactiveFormsModule } from '@angular/forms';
-import { MaterialModule } from '../modules/material.module';
+import { MaterialModule } from 'modules/material.module';
```

Comment:
Just to check ... why did these change even if we didn't move the actual modules?

Reply:
I just refactored it to not use relative imports.